deltacat/catalog/__init__.py

```diff
@@ -1,5 +1,4 @@
 from deltacat.catalog.model.properties import (  # noqa: F401
     CatalogProperties,
     get_catalog_properties,
-    initialize_properties,
 )
```

Comment:
We should also add a corresponding `__all__` section to expose `CatalogProperties`, `get_catalog_properties`, and anything else we intend to expose as direct imports from the `deltacat.catalog` module going forward.